docs/TVM_EP.md

```diff
@@ -201,6 +217,35 @@ po = [dict(executor=tvm_executor_type,
            input_shapes = input_shapes_str)]
 tvm_session = onnxruntime.InferenceSession(model_path, providers=["TvmExecutionProvider"], provider_options=po)
 ```
+
+2. C#
+
+Currently, only precompiled models are supported in C# (see the related section below).
+
+```CSharp
+SessionOptions session_options = new SessionOptions
+{
+  GraphOptimizationLevel = GraphOptimizationLevel.ORT_DISABLE_ALL,
+  LogSeverityLevel = OrtLoggingLevel.ORT_LOGGING_LEVEL_VERBOSE,
+};
+string tvm_ep_options = 
```

Comment:
It is possible to add a comment why this is needed unless it is meant to be removed later?

Reply:
@adelbertc, could you comment?

Reply:
I assume optimization are disable to let tvm use the original graph and not the optimized one. I guess it was already mentioned somewhere. I just meant to recall that information.

Reply:
@KJlaccHoeUM9l plans to remove this call, will update PR

Reply:
Hello @xadupre!
Yes, you are right, this information is indeed mentioned below.
In addition, this parameter is required to use tuning statistics, and in C# this mode is not available (this is described a little higher).
I removed this block of code, we can continue the review.